src/App.css

```diff
@@ -1,59 +1,99 @@
-@import url("https://fonts.googleapis.com/css?family=Kreon|Quicksand");
+@import url('https://fonts.googleapis.com/css?family=Kreon|Quicksand|Pacifico|Lobster|Raleway:600,800');
 
 body {
-  background-color: #c0c0c0;
-  background-image: url("./public/Beer.jpg");
-  opacity: 0.9;
+  background: linear-gradient(to right, #2b344e, #10172b);
+  /* background-image: url("./public/Beer.jpg"); */
+  /* opacity: 0.9; */
 
   /* Full height */
   height: 100%;
 
   /* Center and scale the image nicely */
-  background-position: center;
+  /* background-position: center;
   background-repeat: no-repeat;
-  background-size: cover;
+  background-size: cover; */
 }
```

Comment:
There's already a body selector in `src/index.css`. Perhaps move the above css into `src/index.css`?